python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -334,44 +335,47 @@ def generate_image(self, science, template, difference, center, scale,
         if (source is None) ^ (flags is None):
             raise RuntimeError("Must pass both `source` and `flags` together.")
         if not self.config.use_footprint:
-            extent = lsst.geom.Extent2I(self.config.size, self.config.size)
-            cutout_science = science.getCutout(center, extent)
-            cutout_template = template.getCutout(center, extent)
-            coutout_difference = difference.getCutout(center, extent)
-            size = self.config.size
+            sizes = self.config.sizes
+            cutout_science, cutout_template, cutout_difference, extent = [], [], [], []
+            for i, s in enumerate(sizes):
```

Comment:
`extent` is unused except inside the loop, so all you need here is `for size in sizes:` and then `extent=lsst.geom.Extent2I` below. No need to save the individual extents in a list, since they aren't used.

Reply:
Removed the redundant `extent` list.